helm/korifi/values.yaml

```diff
@@ -44,6 +44,16 @@ api:
       idle: 900
       readHeader: 10
 
+  infoConfig:
+    build: ""
+    description: ""
+    name: ""
+    version: 0
+    minCLIVersion: ""
+    minRecommendedCLIVersion: ""
```

Comment:
Same as above, renaming `minRecommendedCLIVersion` to `recommendedCLIVersion` would probably make sense

Reply:
I've changed it.